LocationList.py

```diff
@@ -824,9 +824,11 @@ def shop_address(shop_id, shelf_id):
     'Collectable': [name for (name, data) in location_table.items() if data[0] == 'Collectable'],
     'BossHeart': [name for (name, data) in location_table.items() if data[0] == 'BossHeart'],
     'CollectableLike': [name for (name, data) in location_table.items() if data[0] in ('Collectable', 'BossHeart', 'GS Token')],
+    'CanSee': [name for (name, data) in location_table.items() if data[0] in ('Collectable, BossHeart', 'GS Token', 'Shop')],
```

Comment:
`'Collectable', 'BossHeart'`

Reply:
I don't think 'CollectableLike' is valid here. I was just pointing out a typo.

Reply:
Hmm, you're probably be right. I'm aware you were pointing out a typo, but I saw CollectableLike used Collectable thinking it was using the group defined before it so thought I could do that for some reason. I'll fix it back to the other way but without the typo.